src/components/ItineraryCalendar.tsx

```
@@ -0,0 +1,140 @@
+// 1. Setup Date Localizer for react-big-calendar
+const locales = { 'en-US': enUS };
+const localizer = dateFnsLocalizer({ format, parse, startOfWeek, getDay, locales });
+const DnDCalendar = withDragAndDrop(Calendar);
+
+export const ItineraryCalendar = () => {
+  // --- STATE ---
+  
+  // Active events on the calendar
+  const [events, setEvents] = useState([
+    {
+      id: 'evt-1',
```

ESLint

[error] Replace `'evt-1'` with `"evt-1"`

(prettier/prettier)